.github/workflows/docker-build.yaml

```diff
@@ -52,6 +52,7 @@ jobs:
             type=ref,event=tag
             type=sha,prefix=git-
             type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
```

Comment:
Thanks for the PR! Would `type=semver,pattern={{major}}.{{minor}}.{{patch}}` also work in this case?

Reply:
That will already happen, no need to add it.

Reply:
Great!